src/utils/bugzilla/chartJs/burndownDataFormatter.js

```diff
@@ -53,7 +53,7 @@ const burndownDataFormatter = (bugSeries, startDate) => {
     bugSeries.forEach(({ bugs, label }, index) => {
         const bugCountPerDay = bugsByCreationDate(bugs, startDate);
         newData.data.datasets.push({
-            ...generateLineChartStyles(SETTINGS.colors[index]),
+            ...generateDatasetStyle(SETTINGS.colors[index]),
```

Comment:
Again, a written standard on what properties are named what would help here; even if they are not used right now; at least they will not change for every feature we will want to add control for.